thirdeye-ui/src/app/pages/anomalies-create-page/anomalies-create-page.component.tsx

```diff
@@ -0,0 +1,214 @@
+import { useGetAlerts } from "../../rest/alerts/alerts.actions";
+import { createAnomaly } from "../../rest/anomalies/anomalies.rest";
+import { Alert } from "../../rest/dto/alert.interfaces";
+import { notifyIfErrors } from "../../utils/notifications/notifications.util";
+import { getErrorMessages } from "../../utils/rest/rest.util";
+import {
+    getAlertsAlertViewPath,
+    getAnomaliesAllPath,
+    getAnomaliesCreatePath,
+    getAnomaliesViewPath,
+} from "../../utils/routes/routes.util";
+import { EditableAnomaly } from "./anomalies-create-page.interfaces";
+
+export const AnomaliesCreatePage: FunctionComponent = () => {
+    const { t } = useTranslation();
+    const navigate = useNavigate();
+    const { notify } = useNotificationProviderV1();
+    const { alerts, getAlerts, status: alertsStatus } = useGetAlerts();
+    const { id: selectedAlertId } = useParams<{ id?: string }>();
+    const [isHelpPanelOpen, setIsHelpPanelOpen] = useState<boolean>(false);
+
+    useEffect(() => {
+        getAlerts();
+    }, []);
+
+    const selectedAlert = useMemo<Alert | null>(() => {
+        if (alerts) {
+            const searchedAlert = alerts.find(
+                (a) => a.id === Number(selectedAlertId)
+            );
+            if (searchedAlert) {
+                return searchedAlert;
+            }
+        }
+
+        return null;
+    }, [alerts, selectedAlertId]);
+
+    const handleCancelClick = (): void => {
+        navigate(getAnomaliesAllPath());
+    };
+
+    const handleAnomalyCreate = (editableAnomaly: EditableAnomaly): void => {
+        createAnomaly(editableAnomaly)
+            .then((data) => {
+                notify(
+                    NotificationTypeV1.Success,
+                    t("message.create-success", { entity: t("label.anomaly") })
+                );
+                // TODO: Remove
+                window.open(getAnomaliesViewPath(data.id), "_blank");
+            })
+            .catch((error) => {
+                notifyIfErrors(
+                    ActionStatus.Error,
+                    getErrorMessages(error),
+                    notify,
+                    t("message.create-error", {
+                        entity: t("label.alert"),
+                    })
+                );
+            });
+    };
+
+    const pageHeaderProps: PageHeaderProps = {
+        title: t("label.report-missed-anomaly"),
+        breadcrumbs: [
+            {
+                link: getAnomaliesAllPath(),
+                label: t("label.anomalies"),
+            },
+            {
+                link: getAnomaliesCreatePath(),
+                label: t("label.create"),
+            },
+            ...(selectedAlert
+                ? [
+                      {
+                          link: getAlertsAlertViewPath(selectedAlert.id),
+                          label: selectedAlert.name,
+                      },
+                  ]
+                : []),
```

Comment:
Let's `if` statement this instead easier to read

Reply:
Updated. Using a simpler `if` statement to conditionally add this link.